### ISSUE TITLE
"Virtualization service (libvirt) is not active" tells me about wrong service

### ISSUE THREAD

**Issue by stefwalter** [cockpit-project/cockpit-machines#1462]
On Fedora 40 (and perhaps elsewhere) the Virtualization Service that the "Virtual machines" tab indicates that:

![image](https://github.com/cockpit-project/cockpit-machines/assets/795070/c7033952-4623-402c-927c-2d89ab910a0a)

However, the service is called `virtqemud`

```
Failed to get libvirt version from the dbus API: 
Object { problem: null, name: "org.libvirt.Error", message: "Failed to connect socket to '/var/run/libvirt/virtqemud-sock': No such file or directory", toString: toString() }
```

and

```
$ sudo systemctl status libvirtd
Unit libvirtd.service could not be found.
$ sudo systemctl status virtqemud
● virtqemud.service - libvirt QEMU daemon
...
```

Version:

```
cockpit-machines-308-1.fc40.noarch
```

**Comment by martinpitt** [comment]
Indeed, this is currently hardcoded in the string. Our integration tests have
```py
    def getLibvirtServiceName(self):
        m = self.machine
        return "libvirtd" if hasMonolithicDaemon(m.image) else "virtqemud"
```
but the runtime code doesn't.

This is a bit awkward to determine efficiently, but if we are on that page there is nothing else to do, so that's fine.

Thanks for your report @stefwalter , nice to see you back!

**Comment by Wotisrv** [comment]
The issue also occurs under Fedora 39 and earlier. The issue only comes up when restarting. It takes a while and then suddenly it's gone and the services are running. Not a problem for me because I rarely restart my server, but it is annoying when the start of the VMs is delayed due to problems with libvirtd or virtqemud.
By the way, libvirt runs as a service after a restart without any problems. Only virtqemud doesn't want to start properly.
